README.md

```diff
@@ -11,23 +11,78 @@ KAGOME is a [Polkadot Host](https://github.com/w3f/polkadot-spec/tree/master/hos
 
 ## Status
 
-- [x] Syncing node
-    - Polkadot, Kusama and Rococo compatibility
-- [x] Validating node
-- [x] Polkadot JS apps support
-- [ ] Parachains support
+![kagome-components-Host drawio-light](https://github.com/soramitsu/kagome/assets/9370151/a39f1c26-329b-4163-a112-7bf6512b6370)
```

Comment:
Why is RocksDb a part of TrieStorage here?

Reply:
HostAPI block is weird.

Reply:
> HostAPI block is weird.

There were not enough place to describe better unfortunately